sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -484,6 +491,14 @@ public static int getMaxRetriesInLocalRegionWhenRemoteRegionPreferred() {
                 MIN_MAX_RETRIES_IN_LOCAL_REGION_WHEN_REMOTE_REGION_PREFERRED);
     }
 
+    public static int getMaxItemSizeForVectorSearch() {
```

Comment:
Should it support environment variables as well?

Reply:
Yes we discussed that in the meeting, for now we are supporting env variables as well. If the customer is facing some issue, the easiest way would be to reconfigure this value using an env variable. As this will be new feature, and we don't have any data on the customer's query patterns.

Reply:
@aayush3011 - but this implementation only allows system properties - not environment variable - I think that was @jeet1995 's point? And I agree this should also include env variable

Reply:
Added the environment variable as well.